projects/step-core/src/lib/directives/test-dataid.directive.ts

```diff
@@ -0,0 +1,20 @@
+import { Directive, effect, ElementRef, inject, input, Input, InputSignal, Renderer2 } from '@angular/core';
+
+@Directive({
+  selector: '[stepTestId]',
+})
+export class TestDataIdDirective {
+  testDataId: InputSignal<string | undefined> = input<string | undefined>();
```

Comment:
Currently now, directive selector and input attributed doesn't link with each other. The valid usage of such directive would be: `<input type="text" stepTestId testDataId="some-field"/>`, 
but we need to be it like: `<input type="text" stepTestId="some-field"/>`, 

To achieve it, provide an alias to the input field.
```suggestion
  testDataId: InputSignal<string | undefined> = input<string | undefined>({ alias: 'stepTestId' });
```